x-pack/plugin/ccr/src/test/java/org/elasticsearch/xpack/ccr/IndexFollowingIT.java

```diff
@@ -427,37 +421,11 @@ public void testFollowIndexMaxOperationSizeInBytes() throws Exception {
         assertTotalNumberOfOptimizedIndexing(resolveFollowerIndex("index2"), 1, numDocs);
     }
 
-    public void testDontFollowTheWrongIndex() throws Exception {
```

Comment:
It is no longer possible to supply a leader index to the resume follow api and therefor this test is no longer needed. Prior this change you could specify a different leader index / cluster than was specified in put follow api or prior follow resume api calls.